java-engine/lib/src/main/java/io/getunleash/engine/UnleashEngine.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
```

Comment:
Is this still true?

Reply:
Good catch! It was generated by Gradle but I fiddled, taken it out, thank you!